NSBrazilConf/NSBrazilConfApp.swift

```diff
@@ -0,0 +1,12 @@
+import SwiftUI
+
+@main
+struct NSBrazilConfApp: App {
+    @UIApplicationDelegateAdaptor private var appDelegate: AppDelegate
+    
+    var body: some Scene {
+        WindowGroup {
+            HomeTabBar(model: FeedViewModel())
```

Comment:
No momento deixei a Home Antiga para manter a retrocompatibilidade com o atual e não gerar confusão

Reply:
Também acho que dá pra remover ela, se a gente vai manter tudo no feature branch, mas a ideia que eu tinha pra isso é que a gente não precisa de uma home separada pra fazer essas coisas porque é tudo server-driven, então em teoria a home nova são só componentes novos, a menos que a gente precise fazer mudanças mais profundas no próprio container. O parser já ignora todos os componentes que ele não consegue identificar, então a gente não vai ter problema com retrocompatibilidade 😄 

A gente tem mudanças de UI pra fazer na home que não seja dentro dos componentes individuais nesse momento?

Reply:
Show de bola! Se vamos manter a lógica de server driven, então a ideia vai ser criar componentes novos! Vou mudar um pouco a abordagem do PR então! ♥️

Reply:
Vamos! Fica muito mais fácil pra gente quebrar o desenvolvimento em pedaços e experimentar coisas novas quando for necessário